src/notebooks/controllers/controllerLoader.ts

```diff
@@ -50,7 +50,7 @@ export class ControllerLoader implements IControllerLoader, IExtensionSyncActiva
         this.loadControllers();
     }
     private loadControllers() {
-        this.loadControllersImpl();
+        this.loadControllersImpl().ignoreErrors();
```

Comment:
Looks like the two following lines for telemetry should be sent .then from the load controllers impl.